WC-Local-Pickup/woocommerce-local-pickup-time
# Adds Pickup Time Required & Local Pickup Link Capabilities

### All Submissions:






### Changes proposed in this Pull Request:



- Updates NPM & Composer dependencies.
- Moves the plugins settings to WooCommerce->Settings->Shipping->Local Pickup Time settings.
- Reformats some of the plugin settings display.
- Adds new settings screenshots.
- Removes checks that prevent loading the Admin class during AJAX calls.
- Adds the ability to change whether the Pickup Time is required on checkout via a plugin setting. Closes #71
- Adds the ability to link Pickup Time to a Local Pickup Shipping Method. Closes #23 / Closes #94
- Changes output of Pickup Time selection. Closes #103 / Closed #104
- Changes the Pickup Time select field options to use a blank entry for the default upon load. Closes #123

### How to test the changes in this Pull Request:

1. Install these changes into a WooCommerce site.
2. Find the new location of the settings as described in the changes.
3. Optionally change the setting that requires the Pickup Time.
4. Observe the impact of this setting on the checkout screen.
5. Optionally change the setting to link Pickup Time to Local Pickup Shipping methods.
6. Enable/disable Pickup Time on Local Pickup shipping methods within Zones.
7. Observe that the Pickup Time is no linked directly to the selection of the Local Pickup shipping method during checkout.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- Moves the plugins settings to WooCommerce->Settings->Shipping->Local Pickup Time settings and changes some formating.
- Adds the ability to change whether the Pickup Time is required on checkout via a plugin setting.
- Adds the ability to link Pickup Time to a Local Pickup Shipping Method.
- Changes output location and formatting of Pickup Time selection on checkout.
- Changes the Pickup Time select field options to use a blank entry as the default upon load.
